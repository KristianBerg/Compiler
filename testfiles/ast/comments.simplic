int f () {
	int a; // aaaaaaaaaaaaaaaa
	a = 1; // +1*3&#Â¤&%#%&
	//asfdasfdasdf
} //asdfaaafsd
//afsdaf
int g () { //asasdf
	//asdf
} //end of file
