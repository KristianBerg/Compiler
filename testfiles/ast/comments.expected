Program
  List
    Func
      IdDecl ID="f"
      List
      Block
        List
          IdDeclStmt
            IdDecl ID="a"
          Assign
            IdUse ID="a"
            Numeral NUM="1"
    Func
      IdDecl ID="g"
      List
      Block
        List
